WatchParty/client/src/components/Profile/Profile.jsx

```diff
@@ -1,28 +1,323 @@
+    /* DUMMY IMPLEMENTATION FOR TESTING */
+    this.setState({
+      userDescription: "Ollie's profile",
+    });
+  }
+
+  fetchToWatchMovies() {
+    /* MAKE BACKEND API CALL TO FETCH TOWATCH LIST WHICH IS AN ARRAY OF MOVIEIDS */
+    /* DUMMY IMPLEMENTATION FOR TESTING */
+    const toWatch = [131631, 131634, 671, 673, 39538, 146233];
+    toWatch.forEach((element) => {
+      theMovieDb.movies.getById(
+        { id: element },
+        (data) => {
+          const response = JSON.parse(data);
+          this.setState((prevState) => ({
+            toWatchMovieData: [...prevState.toWatchMovieData, response],
+          }));
+        },
+        (err) => console.log(err), // eslint-disable-line no-console
+      );
+    });
+    this.setState({ loadingToWatchMovieData: false });
+  }
+
+  fetchWatchedMovieData() {
+    /* MAKE BACKEND API CALL TO FETCH WATCHED LIST WHICH IS AN ARRAY OF MOVIEIDS */
+    /* DUMMY IMPLEMENTATION FOR TESTING */
+    const watched = [299536, 466272, 530915];
+    watched.forEach((element) => {
+      theMovieDb.movies.getById(
+        { id: element },
+        (data) => {
+          const response = JSON.parse(data);
+          this.setState((prevState) => ({
+            watchedMovieData: [...prevState.watchedMovieData, response],
+          }));
+        },
+        (err) => console.log(err), // eslint-disable-line no-console
+      );
+    });
+    this.setState({ loadingWatchedMovieData: false });
+  }
+
+  fetchFollowers() {
+    /* MAKE BACKEND API CALL TO FETCH FOLLOWERS LIST WHICH IS AN ARRAY OF USERIDS */
+    /* CALL USERDETAILS API ON EACH USERID TO GET USER FIRST & LAST NAME */
+    /* DUMMY IMPLEMENTATION FOR TESTING */
+    this.setState({
+      userFollowers: ["Phoebe J.", "Rachel L.", "Sally M.", "John D."],
+      loadingUserFollowers: false,
+    });
+  }
+
+  fetchFollowing() {
+    /* MAKE BACKEND API CALL TO FETCH FOLLOWING USERS LIST WHICH IS AN ARRAY OF USERIDS */
+    /* CALL USERDETAILS API ON EACH USERID TO GET USER FIRST & LAST NAME */
+    /* DUMMY IMPLEMENTATION FOR TESTING */
+    this.setState({
+      userFollowing: ["Jim H.", "Kevin M."],
+      loadingUserFollowing: false,
+    });
   }
 
   render() {
     const { userID } = this.props;
+    const {
+      userName, userDescription, loadingToWatchMovieData, toWatchMovieData, loadingWatchedMovieData,
+      watchedMovieData, loadingUserFollowers, userFollowers, userFollowing, loadingUserFollowing,
+    } = this.state;
     return (
       <>
         {userID === "" ? (<h3>Unable to retrieve profile. Please log in.</h3>)
           :
           (
-            <h3>
-              welcome user
-            </h3>
+            <div className="profile-main-container">
+              <div className="profilepic-name-container">
+                <div className="profilepic-container">
+                  <img
+                    src="/userDefault.png"
+                    width="380px"
```

Comment:
Just a nit using `px` I don't think is as encouraged

Reply:
good catch, not entirely sure why sleep-deprived me used px. Fixed.